src/components/vedtak/VedtakOppsummering.tsx

```diff
@@ -19,24 +23,45 @@ const texts = {
     dagerGjenstar: 'Dager gjenstår',
     dagerBrukt: 'Dager brukt hittil',
     totalSykepengedager: 'Sykepengedager totalt',
+    sykepengegrunnlag: 'Sykepengegrunnlag',
+    manedsinntekt: 'Beregnet månedslønn',
+    arsinntekt: 'Omregnet til årslønn',
 };
 
 interface VedtakOppsummeringProps {
     selectedVedtak: VedtakDTO,
 }
 
-const StyledUndertittel = styled(Undertittel)`
-    margin-top: .5em;
-    margin-bottom: .5em;
-`;
-
-
 const VedtakOppsummering = (vedtakOppsummering: VedtakOppsummeringProps) => {
     const { selectedVedtak } = vedtakOppsummering;
+
+    const [ sykepengegrunnlag, setSykepengegrunnlag ] = useState<string>('-')
+    const [ manedsinntekt, setManedsinntekt ] = useState<string>('-')
+    const [ arsinntekt, setArsinntekt ] = useState<string>('-')
+
+    useEffect(() => {
+        if (selectedVedtak?.vedtak.sykepengegrunnlag) {
+            const calculatedSykepengegrunnlag = Math.floor(selectedVedtak?.vedtak.sykepengegrunnlag)
+            setSykepengegrunnlag(ValutaFormat.format(calculatedSykepengegrunnlag || 0) + ' kr')
+        } else {
+            setSykepengegrunnlag('-')
```

Comment:
Trenger vi disse `else`ene når `'-'` allerede er default verdi?

Reply:
Ja, jeg tror det. For om man f.eks først går på vedtak1 som har en verdi, så bytter til vedtak2 som har null verdi, så må den settes tilbake til default verdi('-').  
Ellers må det endres til beregning uten bruk av `useEffect`

Reply:
Ja, skjønner! 👍